yt/odin/Dockerfile

```diff
@@ -0,0 +1,135 @@
+ARG ROOT="/ytsaurus"
+ARG SOURCE_ROOT="${ROOT}/ytsaurus"
+ARG ODIN_ROOT="${SOURCE_ROOT}/yt/odin"
+ARG BUILD_ROOT="${ROOT}/build"
+ARG PYTHON_ROOT="${ROOT}/python"
+ARG VIRTUAL_ENV="${ROOT}/virtualenv"
+
+ARG PROTOC_VERSION="3.20.1"
+
+ARG ODIN_RUNTIME_ROOT="/var/odin"
+ARG ODIN_CHECKS_DIR="${ODIN_RUNTIME_ROOT}/checks"
+ARG ODIN_CHECKS_DATA_DIR="${ODIN_RUNTIME_ROOT}/checks-data"
+
+FROM ubuntu:jammy as lib-builder
```

Comment:
I do not insist on doing it now, but in future it would be nice to have:
```
# 4. How to introduce your own private checks using the present SDK.
```